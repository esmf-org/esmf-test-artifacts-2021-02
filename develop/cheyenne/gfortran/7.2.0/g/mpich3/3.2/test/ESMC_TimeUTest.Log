20201220 015958.915 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20201220 015958.915 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20201220 015958.915 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20201220 015958.915 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20201220 015958.915 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20201220 015958.915 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20201220 015958.915 INFO             PET0 Running with ESMF Version   : ESMF_8_1_0_beta_snapshot_39-24-gc0c902f0ea
20201220 015958.915 INFO             PET0 ESMF library build date/time: "Dec 20 2020" "01:39:22"
20201220 015958.915 INFO             PET0 ESMF library build location : /glade/scratch/himanshu/scripts_dir/daily/gfortran_cheyenne/esmf
20201220 015958.915 INFO             PET0 ESMF_COMM                   : mpich3
20201220 015958.915 INFO             PET0 ESMF_NETCDF                 : disabled
20201220 015958.915 INFO             PET0 ESMF_PNETCDF                : disabled
20201220 015958.915 INFO             PET0 ESMF_PIO                    : enabled
20201220 015958.915 INFO             PET0 ESMF_YAMLCPP                : enabled
20201220 015958.915 INFO             PET0 ESMF_MOAB                   : enabled
20201220 015958.915 INFO             PET0 Beginning Test, file ESMC_TimeUTest.C, line 52
20201220 015958.915 INFO             PET0 NUMBER_OF_PROCESSORS 4
20201220 015958.915 INFO             PET0 PASS Create ESMC_Calendar object, ESMC_TimeUTest.C, line 61
20201220 015958.915 INFO             PET0 PASS Print ESMC_Calendar object, ESMC_TimeUTest.C, line 69
20201220 015958.915 INFO             PET0 PASS Set a Time, ESMC_TimeUTest.C, line 77
20201220 015958.915 INFO             PET0 PASS Print ESMC_Time object, ESMC_TimeUTest.C, line 85
20201220 015958.915 INFO             PET0 PASS Get a Time, ESMC_TimeUTest.C, line 93
20201220 015958.915 INFO             PET0 PASS Check that TimeZone parameter is set and get correctly, ESMC_TimeUTest.C, line 102
20201220 015958.915 INFO             PET0 PASS Check that ESMC_Calendar parameter is set and get correctly, ESMC_TimeUTest.C, line 110
20201220 015958.915 INFO             PET0 PASS Destroy ESMC_Calendar object, ESMC_TimeUTest.C, line 120
20201220 015958.915 INFO             PET0 Ending Test, file ESMC_TimeUTest.C, line 124
20201220 015958.915 INFO             PET0 Finalizing ESMF
20201220 015958.915 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20201220 015958.915 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20201220 015958.915 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20201220 015958.915 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20201220 015958.915 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20201220 015958.915 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20201220 015958.915 INFO             PET1 Running with ESMF Version   : ESMF_8_1_0_beta_snapshot_39-24-gc0c902f0ea
20201220 015958.915 INFO             PET1 ESMF library build date/time: "Dec 20 2020" "01:39:22"
20201220 015958.915 INFO             PET1 ESMF library build location : /glade/scratch/himanshu/scripts_dir/daily/gfortran_cheyenne/esmf
20201220 015958.915 INFO             PET1 ESMF_COMM                   : mpich3
20201220 015958.915 INFO             PET1 ESMF_NETCDF                 : disabled
20201220 015958.915 INFO             PET1 ESMF_PNETCDF                : disabled
20201220 015958.915 INFO             PET1 ESMF_PIO                    : enabled
20201220 015958.915 INFO             PET1 ESMF_YAMLCPP                : enabled
20201220 015958.915 INFO             PET1 ESMF_MOAB                   : enabled
20201220 015958.915 INFO             PET1 Beginning Test, file ESMC_TimeUTest.C, line 52
20201220 015958.915 INFO             PET1 NUMBER_OF_PROCESSORS 4
20201220 015958.915 INFO             PET1 PASS Create ESMC_Calendar object, ESMC_TimeUTest.C, line 61
20201220 015958.915 INFO             PET1 PASS Print ESMC_Calendar object, ESMC_TimeUTest.C, line 69
20201220 015958.915 INFO             PET1 PASS Set a Time, ESMC_TimeUTest.C, line 77
20201220 015958.915 INFO             PET1 PASS Print ESMC_Time object, ESMC_TimeUTest.C, line 85
20201220 015958.915 INFO             PET1 PASS Get a Time, ESMC_TimeUTest.C, line 93
20201220 015958.915 INFO             PET1 PASS Check that TimeZone parameter is set and get correctly, ESMC_TimeUTest.C, line 102
20201220 015958.916 INFO             PET1 PASS Check that ESMC_Calendar parameter is set and get correctly, ESMC_TimeUTest.C, line 110
20201220 015958.916 INFO             PET1 PASS Destroy ESMC_Calendar object, ESMC_TimeUTest.C, line 120
20201220 015958.916 INFO             PET1 Ending Test, file ESMC_TimeUTest.C, line 124
20201220 015958.916 INFO             PET1 Finalizing ESMF
20201220 015958.915 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20201220 015958.915 INFO             PET2 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20201220 015958.915 INFO             PET2 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20201220 015958.915 INFO             PET2 !!! FOR PRODUCTION RUNS, USE:                      !!!
20201220 015958.915 INFO             PET2 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20201220 015958.915 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20201220 015958.915 INFO             PET2 Running with ESMF Version   : ESMF_8_1_0_beta_snapshot_39-24-gc0c902f0ea
20201220 015958.915 INFO             PET2 ESMF library build date/time: "Dec 20 2020" "01:39:22"
20201220 015958.915 INFO             PET2 ESMF library build location : /glade/scratch/himanshu/scripts_dir/daily/gfortran_cheyenne/esmf
20201220 015958.915 INFO             PET2 ESMF_COMM                   : mpich3
20201220 015958.915 INFO             PET2 ESMF_NETCDF                 : disabled
20201220 015958.915 INFO             PET2 ESMF_PNETCDF                : disabled
20201220 015958.915 INFO             PET2 ESMF_PIO                    : enabled
20201220 015958.915 INFO             PET2 ESMF_YAMLCPP                : enabled
20201220 015958.915 INFO             PET2 ESMF_MOAB                   : enabled
20201220 015958.915 INFO             PET2 Beginning Test, file ESMC_TimeUTest.C, line 52
20201220 015958.915 INFO             PET2 NUMBER_OF_PROCESSORS 4
20201220 015958.915 INFO             PET2 PASS Create ESMC_Calendar object, ESMC_TimeUTest.C, line 61
20201220 015958.915 INFO             PET2 PASS Print ESMC_Calendar object, ESMC_TimeUTest.C, line 69
20201220 015958.915 INFO             PET2 PASS Set a Time, ESMC_TimeUTest.C, line 77
20201220 015958.916 INFO             PET2 PASS Print ESMC_Time object, ESMC_TimeUTest.C, line 85
20201220 015958.916 INFO             PET2 PASS Get a Time, ESMC_TimeUTest.C, line 93
20201220 015958.916 INFO             PET2 PASS Check that TimeZone parameter is set and get correctly, ESMC_TimeUTest.C, line 102
20201220 015958.916 INFO             PET2 PASS Check that ESMC_Calendar parameter is set and get correctly, ESMC_TimeUTest.C, line 110
20201220 015958.916 INFO             PET2 PASS Destroy ESMC_Calendar object, ESMC_TimeUTest.C, line 120
20201220 015958.916 INFO             PET2 Ending Test, file ESMC_TimeUTest.C, line 124
20201220 015958.916 INFO             PET2 Finalizing ESMF
20201220 015958.915 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20201220 015958.915 INFO             PET3 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20201220 015958.915 INFO             PET3 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20201220 015958.915 INFO             PET3 !!! FOR PRODUCTION RUNS, USE:                      !!!
20201220 015958.915 INFO             PET3 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20201220 015958.915 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20201220 015958.915 INFO             PET3 Running with ESMF Version   : ESMF_8_1_0_beta_snapshot_39-24-gc0c902f0ea
20201220 015958.915 INFO             PET3 ESMF library build date/time: "Dec 20 2020" "01:39:22"
20201220 015958.915 INFO             PET3 ESMF library build location : /glade/scratch/himanshu/scripts_dir/daily/gfortran_cheyenne/esmf
20201220 015958.915 INFO             PET3 ESMF_COMM                   : mpich3
20201220 015958.915 INFO             PET3 ESMF_NETCDF                 : disabled
20201220 015958.915 INFO             PET3 ESMF_PNETCDF                : disabled
20201220 015958.915 INFO             PET3 ESMF_PIO                    : enabled
20201220 015958.915 INFO             PET3 ESMF_YAMLCPP                : enabled
20201220 015958.915 INFO             PET3 ESMF_MOAB                   : enabled
20201220 015958.915 INFO             PET3 Beginning Test, file ESMC_TimeUTest.C, line 52
20201220 015958.915 INFO             PET3 NUMBER_OF_PROCESSORS 4
20201220 015958.915 INFO             PET3 PASS Create ESMC_Calendar object, ESMC_TimeUTest.C, line 61
20201220 015958.915 INFO             PET3 PASS Print ESMC_Calendar object, ESMC_TimeUTest.C, line 69
20201220 015958.915 INFO             PET3 PASS Set a Time, ESMC_TimeUTest.C, line 77
20201220 015958.915 INFO             PET3 PASS Print ESMC_Time object, ESMC_TimeUTest.C, line 85
20201220 015958.915 INFO             PET3 PASS Get a Time, ESMC_TimeUTest.C, line 93
20201220 015958.915 INFO             PET3 PASS Check that TimeZone parameter is set and get correctly, ESMC_TimeUTest.C, line 102
20201220 015958.915 INFO             PET3 PASS Check that ESMC_Calendar parameter is set and get correctly, ESMC_TimeUTest.C, line 110
20201220 015958.915 INFO             PET3 PASS Destroy ESMC_Calendar object, ESMC_TimeUTest.C, line 120
20201220 015958.915 INFO             PET3 Ending Test, file ESMC_TimeUTest.C, line 124
20201220 015958.915 INFO             PET3 Finalizing ESMF
Sun Dec 20 05:39:47 MST 2020
