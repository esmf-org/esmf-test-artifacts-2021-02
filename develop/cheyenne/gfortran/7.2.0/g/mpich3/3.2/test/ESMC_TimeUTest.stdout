Beginning Test, file ESMC_TimeUTest.C, line 52
NUMBER_OF_PROCESSORS 4
After CalendarCreate rc = 0 
Beginning Test, file ESMC_TimeUTest.C, line 52
NUMBER_OF_PROCESSORS 4
PASS Create ESMC_Calendar object, ESMC_TimeUTest.C, line 61
After CalendarCreate rc = 0 
Calendar -------------------------------
name = Gregorian
calkindflag = Gregorian
daysPerMonth = 31 28 31 30 31 30 31 31 30 31 30 31 PASS Create ESMC_Calendar object, ESMC_TimeUTest.C, line 61

monthsPerYear = 12
secondsPerDay = 86400
secondsPerYear = 31536000
Calendar -------------------------------
name = Gregorian
calkindflag = Gregorian
daysPerMonth = 31 daysPerYear = 365
daysPerYeardN = 0
daysPerYeardD = 1
end Calendar ---------------------------

28 31 30 31 30 31 31 30 31 30 31 
monthsPerYear = 12
secondsPerDay = 86400
secondsPerYear = 31536000
daysPerYear = 365
daysPerYeardN = 0
daysPerYeardD = 1
end Calendar ---------------------------

PASS Print ESMC_Calendar object, ESMC_TimeUTest.C, line 69
PASS Print ESMC_Calendar object, ESMC_TimeUTest.C, line 69
Beginning Test, file ESMC_TimeUTest.C, line 52
NUMBER_OF_PROCESSORS 4
After CalendarCreate rc = 0 
PASS Create ESMC_Calendar object, ESMC_TimeUTest.C, line 61
PASS Set a Time, ESMC_TimeUTest.C, line 77
Calendar -------------------------------
name = Gregorian
calkindflag = Gregorian
daysPerMonth = 31 Time -----------------------------------
PASS Set a Time, ESMC_TimeUTest.C, line 77
28 31 30 31 30 31 31 30 31 30 31 
Time -----------------------------------
monthsPerYear = 12
secondsPerDay = 86400
secondsPerYear = 31536000
daysPerYear = 365
daysPerYeardN = 0
daysPerYeardD = 1
end Calendar ---------------------------

2006-01-01T00:00:00
end Time -------------------------------

2006-01-01T00:00:00
end Time -------------------------------

PASS Print ESMC_Calendar object, ESMC_TimeUTest.C, line 69
PASS Print ESMC_Time object, ESMC_TimeUTest.C, line 85
Beginning Test, file ESMC_TimeUTest.C, line 52
NUMBER_OF_PROCESSORS 4
PASS Print ESMC_Time object, ESMC_TimeUTest.C, line 85
After CalendarCreate rc = 0 
PASS Set a Time, ESMC_TimeUTest.C, line 77
PASS Get a Time, ESMC_TimeUTest.C, line 93
Time -----------------------------------
PASS Check that TimeZone parameter is set and get correctly, ESMC_TimeUTest.C, line 102
PASS Create ESMC_Calendar object, ESMC_TimeUTest.C, line 61
PASS Check that ESMC_Calendar parameter is set and get correctly, ESMC_TimeUTest.C, line 110
Calendar -------------------------------
name = Gregorian
calkindflag = Gregorian
daysPerMonth = 31 28 31 30 31 30 2006-01-01T00:00:00
end Time -------------------------------

PASS Get a Time, ESMC_TimeUTest.C, line 93
31 31 30 31 30 31 
monthsPerYear = 12
secondsPerDay = 86400
secondsPerYear = 31536000
PASS Print ESMC_Time object, ESMC_TimeUTest.C, line 85
PASS Destroy ESMC_Calendar object, ESMC_TimeUTest.C, line 120
daysPerYear = 365
daysPerYeardN = 0
daysPerYeardD = 1
end Calendar ---------------------------

PASS Check that TimeZone parameter is set and get correctly, ESMC_TimeUTest.C, line 102
Ending Test, file ESMC_TimeUTest.C, line 124
PASS Print ESMC_Calendar object, ESMC_TimeUTest.C, line 69
PASS Check that ESMC_Calendar parameter is set and get correctly, ESMC_TimeUTest.C, line 110
PASS Get a Time, ESMC_TimeUTest.C, line 93
PASS Check that TimeZone parameter is set and get correctly, ESMC_TimeUTest.C, line 102
PASS Destroy ESMC_Calendar object, ESMC_TimeUTest.C, line 120
PASS Check that ESMC_Calendar parameter is set and get correctly, ESMC_TimeUTest.C, line 110
PASS Set a Time, ESMC_TimeUTest.C, line 77
Time -----------------------------------
Ending Test, file ESMC_TimeUTest.C, line 124
PASS Destroy ESMC_Calendar object, ESMC_TimeUTest.C, line 120
2006-01-01T00:00:00
end Time -------------------------------

Ending Test, file ESMC_TimeUTest.C, line 124
PASS Print ESMC_Time object, ESMC_TimeUTest.C, line 85
PASS Get a Time, ESMC_TimeUTest.C, line 93
PASS Check that TimeZone parameter is set and get correctly, ESMC_TimeUTest.C, line 102
PASS Check that ESMC_Calendar parameter is set and get correctly, ESMC_TimeUTest.C, line 110
PASS Destroy ESMC_Calendar object, ESMC_TimeUTest.C, line 120
Ending Test, file ESMC_TimeUTest.C, line 124
 PET 1 Test Elapsed Time  0.840000 msec.
 PET 3 Test Elapsed Time  0.926000 msec.
 PET 2 Test Elapsed Time  0.983000 msec.
 PET 0 Test Elapsed Time  0.820000 msec.
Tue Feb  9 08:10:25 MST 2021
